packages/roosterjs-editor-api/lib/table/insertTable.ts

```diff
@@ -30,6 +30,7 @@ export default function insertTable(
             tr.appendChild(td);
             td.appendChild(document.createElement('br'));
             td.style.width = getTableCellWidth(columns);
+            td.style.height = '20px';

```

Comment:
We should not rely on this value to do resize. It is possible that user paste some table which doesn't have a height style. We can calcuate the height at runtime, but not read its style value.